sw/device/silicon_creator/mask_rom/mask_rom.c

```diff
@@ -192,6 +193,23 @@ static rom_error_t mask_rom_verify(const manifest_t *manifest,
                               flash_exec);
 }
 
+/* These symbols are defined in
+* `opentitan/sw/device/silicon_creator/mask_rom/mask_rom.ld`, and describes the
+* location of the flash header.
+*/
+extern char _rom_ext_virtual_address[];
+extern char _rom_ext_virtual_size[];
```

Comment:
Is there a reason why we want to unlock the whole rom_ext area instead of the area used by the rom_ext image (as given by `manifest->length`)?

Reply:
I think that the main reason is that the size must be a power of two (NAPOT), then there's a great change of the `manifest->length` ending up being rounded to the max size anyway.

Reply:
Makes sense, I forgot that entire eFlash is unlocked for read anyway.